src/app.js

```diff
@@ -535,12 +547,6 @@ function run () {
     compilationMessage: (message, container, options) => {
       renderer.error(message, container, options)
     },
-    currentCompiledSourceCode: () => {
```

Comment:
If this is still needed it should still work.

Reply:
this was used at some point for `target`